mws/apis/easyship.py

```diff
@@ -43,18 +50,25 @@ def list_pickup_slots(
         package_weight=0,
         package_weight_uom="g",
     ):
-        data = {
-            "Action": "ListPickupSlots",
-            "MarketplaceId": marketplace_id,
-            "AmazonOrderId": amazon_order_id,
-            "PackageDimensions.Width": str(package_width),
-            "PackageDimensions.Length": str(package_length),
-            "PackageDimensions.Height": str(package_height),
-            "PackageDimensions.Unit": package_dimensions_uom,
-            "PackageWeight.Unit": package_weight_uom,
-            "PackageWeight.Value": str(package_weight),
-        }
-        return self.make_request(data)
+        """Returns time slots for Amazon Easy Ship package pickup,
+        based on the package dimensions and package weight that the seller specifies.
+
+        Docs:
+        http://docs.developer.amazonservices.com/en_IN/easy_ship/EasyShip_ListPickupSlots.html
+        """
+        return self.make_request(
```

Comment:
This needs viewing in full - diff looks odd.

Reply:
Changes like this one are part a change to `make_request`'s signature and part cleanup. First arg is now the action, with second arg, `params` (not named in this case), being the dict of parameters.

In cases where the dictionary `data` didn't require any `.update()` calls to add in enumerations or sub-keys, I opted to call the method directly with the dict literal, without saving it as `data` first.

In hindsight, I should probably go around and at the named `params` arg into the method call for clarity.

Reply:
ah its same as my comments below - the diff view threw me on this one for some reason, all good.